src/scipp/plt/model.py

```diff
@@ -75,10 +81,18 @@ def node(func, *args, **kwargs):
     partialized = partial(func, *args, **kwargs)
 
     def make_node(*args, **kwargs):
-        return Node(partialized, *args, **kwargs)
+        out = Node(partialized, *args, **kwargs)
+        out.label = f"Node: {out.func.func.__name__}"
+        if out.func.args:
+            out.label += f", {out.func.args}"
+        if out.func.keywords:
+            out.label += f", {out.func.keywords}"
+        return out
```

Comment:
Is the label required outside of the graph renderer? If not, it should not be stored in the node object.

Reply:
It is only used in the graph renderer (for now). But I don't really see how we could do it otherwise if having a cleaner graph as above is something we want to keep.
We can't distinguish between a `Node` made from `input_node` or `widget_node` after they have been created. And it's quite useful to be able to see that information on the graph.

Reply:
Other possibility is to make subclasses of `Node` for `InputNode` and `WidgetNode`, but again this would only be for the graph, so it would not solve the issue...

Reply:
Ok I removed the `label` from the `Node`. Graph now looks like this
![Screenshot at 2022-05-24 12-08-58](https://user-images.githubusercontent.com/39047984/170007660-cff23485-f1c3-4331-ada5-70a1e8da8598.png)